sale_payment_term_interest/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Sales Payment Term Interests
+============================
+
+This module allows to compute interest fees based on payment terms and
+to add a line in the sales orders with the computed amount.
+
+Installation
+============
+
+To install this module, you need to:
+
+ * Install the `sale_payment_term_interest` module is the only things to do.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+ * Go to *Invoicing > Configuration > Payment Terms*
+ * On the lines of the payment terms, you can configure an interest
+   rate. It is based on the number of days.
```

Comment:
I find this strange, but I'm don't know the general practice here. Most of the time I see an annual interest rate, which is divided by 12 to get a monthly rate (and then by 30 to get a daily rate if needed).

Is this meant to be used in a B2B or a B2C context?